pkgs/top-level/aliases.nix

```diff
@@ -603,6 +603,7 @@ mapAliases ({
   cstore_fdw = postgresqlPackages.cstore_fdw;
   pg_hll = postgresqlPackages.pg_hll;
   pg_cron = postgresqlPackages.pg_cron;
+  pg_rational = postgresqlPackages.pg_rational;
```

Comment:
It's not necessary to add an alias to `postgresqlPackages` here.
```suggestion
```
Those `postgresqlPackages` aliases are kept only for backwards compatibility